### PR TITLE
agent store: add update queue methods

### PR DESCRIPTION
**Pull request by efim-poberezkin**

### CODE REVIEW

File: src/Simplex/Messaging/Agent.hs [hunk 1]
```diff
@@ -272,12 +272,12 @@ sendConfirmation ::
   SendQueue ->
   SMP.SenderKey ->
   m ()
-sendConfirmation c SendQueue {server, sndId} senderKey = do
+sendConfirmation c sndQ@SendQueue {server, sndId} senderKey = do
```
Comment (epoberezkin) [inline src/Simplex/Messaging/Agent.hs:275]: rcvQ and sndQ are used I think - doesn't he warn about shadowing?

Comment (epoberezkin) [reply src/Simplex/Messaging/Agent.hs:275]: can be just `rq` or `sq`

Comment (efim-poberezkin) [reply src/Simplex/Messaging/Agent.hs:275]: I don't get warnings.. Where are they used?

Comment (efim-poberezkin) [reply src/Simplex/Messaging/Agent.hs]: w/e, changed anyway

**Comment by epoberezkin** [comment]
that's the bug I fixed in my PR - I've just re-run the tests

**Review by epoberezkin — APPROVED** [review]

**Pull request merged**